t/t1092-sparse-checkout-compatibility.sh

```diff
@@ -1319,7 +1319,7 @@ test_expect_success 'stash -u outside sparse checkout definition' '
 	test_all_match git status --porcelain=v2 &&
 
 	test_all_match git stash pop -q &&
-	test_all_match git status --porcelain=v2
+	test_sparse_match git status --porcelain=v2
```

Comment:
The line

```
You are in a sparse checkout with 72% of tracked files present.
```

only appears in `git status`, not `git status --porcelain=v2` (hence why we use it so often in this test). There must be something different happening in your environment.

Reply:
Ah, that's good to know! I re-ran this morning without the change, and the test passed, so not sure what was going on there. At any rate, I've removed that !fixup commit.